maintainers/maintainer-list.nix

```diff
@@ -8621,6 +8621,10 @@
       fingerprint = "FEF0 AE2D 5449 3482 5F06  40AA 186A 1EDA C5C6 3F83";
     }];
   };
+  mightyiam = {
+    name = "Shahar Dawn Or";
+    github = "mightyiam";
```

Comment:
you also need githubid and email.
I can't remember off the top of my head how to get that, but googling should help you there.

Reply:
also small thing, usually maintainer list is added when someone becomes a maintainer, are you interested in maintaining any of the packages ? (it doesn't need to be lemmy, it can be something else).